README.md

```diff
@@ -24,39 +27,48 @@ All other Secondary Pi-holes that you wish to keep synchronized use `pihole-clou
 
 **Before proceeding, verify that your Primary and *all* Secondary Pi-holes are running Pi-hole v5 or later.**
 
+## Docker Support
+
+This script supports docker containers running the `pihole/pihole` image.
+The container must be started when the script is run. The `pihole-cloudsync` should work out of the box with no changes required.
+
 # Setup
-Prior to running `pihole-cloudsync`, you must first create a new dedicated Git respository to store your lists, then clone that new repository to all Pi-holes (both Primary and Secondary) that you wish to keep in sync. The easiest way to do that is to fork my own <a target="_blank" href="https://github.com/stevejenkins/my-pihole-lists">`my-pihole-lists`</a> GitHub repository. Don't worry if the example lists in that repo are different than yours. You'll overwrite your forked version of the repo with your own Pi-hole lists the first time you run `pihole-cloudsync` in **Push** mode.
+
+Prior to running `pihole-cloudsync`, you must first create a new dedicated Git repository to store your lists, then clone that new repository to all Pi-holes (both Primary and Secondary) that you wish to keep in sync. The easiest way to do that is to fork my own <a target="_blank" href="https://github.com/stevejenkins/my-pihole-lists">`my-pihole-lists`</a> GitHub repository. Don't worry if the example lists in that repo are different than yours. You'll overwrite your forked version of the repo with your own Pi-hole lists the first time you run `pihole-cloudsync` in **Push** mode.
 
 **On GitHub**
+
 1. Sign into GitHub.
-2. Go to https://github.com/stevejenkins/my-pihole-lists.
-3. Press **Fork**.
-4. *Optional:* If you wish to make your forked version of the repo private, press **Settings**, scroll down to the **Danger Zone**, then press **Make private**.
-5. On your new repo's main page, press the **Clone or download** button and copy the **Clone with HTTPS** link to your clipboard.
+1. Go to https://github.com/stevejenkins/my-pihole-lists.
+1. Press **Fork**.
+1. *Optional:* If you wish to make your forked version of the repo private, press **Settings**, scroll down to the **Danger Zone**, then press **Make private**.
+1. On your new repo's main page, press the **Clone or download** button and copy the **Clone with HTTPS** link to your clipboard.
 
 **On your Primary Pi-hole device**
+
 1. Install Git (on Raspbian/Debian do `sudo apt-get install git`).
-2. Do `cd /usr/local/bin`.
-3. Install `pihole-cloudsync` with `sudo git clone https://github.com/stevejenkins/pihole-cloudsync.git`.
-4. Create your private local Git repo with `sudo git clone https://github.com/<yourusername>/my-pihole-lists.git` (paste the URL you copied from GitHub).
-5. If you're using a repo name other than `my-pihole-lists`, edit `/usr/local/bin/pihole-cloudsync/pihole-cloudsync` and edit the `personal_git_dir` variable to match your local Git repo location.
```

Comment:
Removed this instruction here and in the next section. Changed to auto-numbered lists.